lib/test.js

```diff
@@ -0,0 +1,261 @@
+    return {
+      passed: false,
+      message: getFailErrorMessage({
+        ruleName: rule.ruleName,
+        functionName: rule.rule.functionName,
+        functionsFile: rule.functionsFile,
+        exampleTitle: example.meta.title,
+        exampleFile: example.file,
+        expected: example.meta.error,
+        actual: '',
+        target: target || example.meta.target,
+        type,
+      }),
+    };
+  }
+
+  // no error expected but got one
+  if ((_.isArray(VkResult) && VkResult.length !== 0) || vkError) {
+    return {
+      passed: false,
+      message: getFailErrorMessage({
+        ruleName: rule.ruleName,
+        functionName: rule.rule.functionName,
+        functionsFile: rule.functionsFile,
+        exampleTitle: example.meta.title,
+        exampleFile: example.file,
+        expected: '',
+        actual,
+        target: target || example.meta.target,
+        type,
+      }),
+    };
+  }
+
+  // no error expected , got no error
+  return {
+    passed: true,
+    message: getPassedErrorMessage({
+      ruleName: rule.ruleName,
+      exampleTitle: example.meta.title,
+      type,
+      target: target || example.meta.target,
+    }),
+  };
+}
+
+let atLeastOneFailed = false;
+const failed = [];
+
+
+helpers.getApiaryToken((err, apiaryToken) => {
+  if (err) {
+    console.error(err.message || err);
+    process.exit(1);
+  }
+
+  function getTargets(example) {
+    let targets = _.get(example, 'meta.target', []);
+
+    if (!_.isArray(targets)) {
+      targets = [targets];
+    }
+
+    return targets;
+  }
+
+  const tests = [];
+
+  _.each(_.get(styleGuides, 'rules', []), (rule) => {
+    _.each(_.get(rule, 'examples.good', []), (example) => {
+      _.each(getTargets(example), (target) => {
+        tests.push({
+          rule,
+          example,
+          type: 'good',
+          target,
+        });
+      });
+    });
+    _.each(_.get(rule, 'examples.bad', []), (example) => {
+      _.each(getTargets(example), (target) => {
+        tests.push({
+          rule,
+          example,
+          type: 'bad',
+          target,
+        });
+      });
+    });
+  });
+
+
+  let currentRule = '';
+
+  async.eachSeries(tests, (test, next) => {
+    helpers.testExample({
+      functions,
+      example: test.example,
+      functionName: _.get(test, 'rule.rule.functionName'),
+      minim: _.get(test, 'rule.rule.minim'),
+      target: test.target,
+      apiaryToken,
+    }, (error, results, target) => {
+      if (error) {
+        return next(error);
+      }
+      const result = getResult({
+        VkResult: results,
+        rule: test.rule,
+        example: test.example,
+        type: test.type,
+        target,
+      });
+
+      if (!result.passed) {
+        atLeastOneFailed = true;
+        failed.push(result.message);
+      }
+
+      if (currentRule !== test.rule.ruleName) {
+        currentRule = test.rule.ruleName;
+        console.log('');
+      }
+
+      console.log(result.message);
```

Comment:
This outputs the whole webpack bundle, so it's impossible to read

Reply:
how did you achieve it ?
it should be something like ```
[❌] FAILED: `sample_rule` failed for `bad` example: `title for 'should fail' example` (target: `Response_Body`)
  example: styleguides/sample_rule/examples/bad/sample-bad-example/api-description-document
  function: someFunction in styleguides/sample_rule/functions.js
  - expected + actual 
    - ooooo nooooooo error!
    + ooooo nooooooo errsssor!
```

Reply:
```
[❌] FAILED: `delete_no_body` failed for `bad` example: `Body pasyload in response` (target: `Action`)
  example: styleguides/delete_no_body/examples/bad/sample-bad-example/api-description-document
  function: validateDeleteNoBody in styleguides/delete_no_body/functions.js
  - expected + actual
    - DELETE request must not have a body.
    + Validation error: Invalid rules or/and functions
 Unexpected token (174:39) in

/*
Sample rule title
Intent of the rule/description

@targets: Resource_URI_Template, Response_Body
@minim: true
*/
function someFunction(data) {
  return functions.someFunction_webpack(data);
}

/*
Only British spelling
Validates if there is no american spelling in JSON object bodies

@targets: Request_Body, Response_Body
@minim: true
*/
function validateBodyBritishSpelling(data) {
  return functions.validateBodyBritishSpelling_webpack(data);
}

// …
```

---

Line ~174 looks like this

```js
/******/
/******/ 	// define getter function for harmony exports
/******/ 	__webpack_require__.function d(exports, name, getter) { 
/******/ 		if(!__webpack_require__.o(exports, name)) { // 174
/******/ 			Object.defineProperty(exports, name, {
/******/ 				configurable: false,
/******/ 				enumerable: true,
/******/ 				get: getter
/******/ 			});
/******/ 		}
/******/ 	};
/******/
```

Reply:
ah. yup - it is calling old VK (HH) which is not able to process new rules = message is correct - it outputs provided functions